rococo/repositories/surreal_db_repository.py

```diff
@@ -0,0 +1,31 @@
+import json
+from typing import Type
+
+from rococo.data.surrealdb import SurrealDbAdapter
```

Comment:
@caeser1996 Can we shorten the imports to just import from package?

i. e `roco.data import SurrealDbAdapter`

Reply:
from rococo.data import SurrealDbAdapter
from rococo.messaging import MessageAdapter
from rococo.models import VersionedModel
from rococo.repositories import BaseRepository

Like this ?

Reply:
Yes

Reply:
Done sir

Reply:
Yes, I've put to my Todo list too, bc I've seen some of those long ones still hanging around ;)